libs/execution_context/ebpf_core.c

```diff
@@ -201,22 +204,27 @@ ebpf_core_initiate()
 
     _ebpf_global_helper_program_info.count_of_program_type_specific_helpers = ebpf_core_helper_functions_count;
     _ebpf_global_helper_program_info.program_type_specific_helper_prototype = ebpf_core_helper_function_prototype;
-    return_value = ebpf_provider_load(
-        &_ebpf_global_helper_function_provider_context,
+
```

Comment:
suggest move nmr registration code in a separate function in this file.

Reply:
Simplified the code to a single NMR call.